### DISCUSSION TITLE
Can I automatically insert data into one collection on creation of record in another?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [pocketbase/pocketbase#1325] [Q&A] [answered]
I have the collections `User` and `Streamer` and would like to create a record in the Streamer collection whenever a new User is made with a one-to-one relationship between the two records.

Does Pocketbase have any utilities to do this automatically?

**Comment by ganigeorgiev** [comment, marked as answer]
No there is no automatic way to do it. If you are embedding PocketBase in your Go app, you can use the [event hooks](https://pocketbase.io/docs/event-hooks/) (soon there will be also an option to bind to the hooks with external js).

You could also check #317.
